modules/admin-ui-picard/app/src/thunks/aclDetailsThunks.js

```diff
@@ -0,0 +1,110 @@
+import {loadAclDetailsFailure, loadAclDetailsInProgress, loadAclDetailsSuccess} from "../actions/aclDetailsActions";
+import axios from "axios";
+import {addNotification} from "./notificationThunks";
+import {prepareAccessPolicyRulesForPost} from "../utils/resourceUtils";
+
+// fetch details about a certain acl from server
+export const fetchAclDetails = aclId => async dispatch => {
+    try {
+        dispatch(loadAclDetailsInProgress());
+
+        let data = await axios.get(`admin-ng/acl/${aclId}`);
+
+        let aclDetails = await data.data;
+
+        let acl = aclDetails.acl;
+        let transformedAcls = [];
+
+        // transform policies for further use
+        for (let i = 0; acl.ace.length > i; i++) {
+            if (transformedAcls.find(rule => rule.role === acl.ace[i].role)) {
+                for (let j = 0; transformedAcls.length > j; j++) {
+                    // only update entry for policy if already added with other action
+                    if (transformedAcls[j].role === acl.ace[i].role) {
+                        if (acl.ace[i].action === "read") {
+                            transformedAcls[j] = {
+                                ...transformedAcls[j],
+                                write: acl.ace[i].allow
```

Comment:
should be: `read: acl.ace[i].allow` not `write: acl.ace[i].allow`
```suggestion
                                read: acl.ace[i].allow
```